odl/solvers/advanced/proximal_operators.py

```diff
@@ -613,6 +652,25 @@ def proximal_l2(space, lam=1, g=None):
 
     Notes
     -----
+    The functional :math:`F` is given by
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_2
+
+    For a step size :math:`\\sigma`, the proximal operator of :math:`\\sigma F`
+    is given by
+
+        .. math::
+
+            prox[\\sigma F](y) = \\left\{ \\begin{array}{ll}
+            \\frac{1 - c}{||y-g||} \\cdot y  + c \cdot g
+            & \\quad  \\text{if } c < g, \\\\
+            g & \\text{else},
```

Comment:
You have a quad before the if statement, but not the else, so this looks weird when typeset.